modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/Public/CmsKit/Blogs/BlogPost.cshtml

```diff
@@ -16,94 +16,133 @@
 
 @inject IMarkdownToHtmlRenderer MarkdownRenderer
 
+
+@{
+    string dummyImageSource = "https://dummyimage.com/1280x720/a3a3a3/fff.png?text=" + Model.BlogPost.Title;
+    var isScrollIndexEnabled = GlobalFeatureManager.Instance.IsEnabled<BlogPostScrollIndexFeature>();
+}
+
 @section styles{
-<abp-abp-style-bundle>
-    <abp-style src="/Pages/Public/CmsKit/Blogs/blogPost.css" />
-    <abp-style type="typeof(HighlightJsStyleContributor)" />
-</abp-abp-style-bundle>
+     @if (isScrollIndexEnabled)
+     {
+         <abp-abp-style-bundle>
+             <abp-style src="/Pages/Public/CmsKit/Blogs/bootstrap-toc.css"/>
+         </abp-abp-style-bundle>
+     }
+    <abp-abp-style-bundle>
+        <abp-style src="/Pages/Public/CmsKit/Blogs/blogPost.css"/>
+        <abp-style type="typeof(HighlightJsStyleContributor)"/>
+    </abp-abp-style-bundle>
 }
 
 @section scripts{
-<abp-script-bundle>
-    <abp-script type="typeof(HighlightJsScriptContributor)" />
-    <abp-script src="/Pages/Public/CmsKit/highlightOnLoad.js" />
-</abp-script-bundle>
-}
+    @if (isScrollIndexEnabled)
+    {
+        <abp-script-bundle>
+            <abp-style src="/Pages/Public/CmsKit/Blogs/bootstrap-toc.js"/>
+        </abp-script-bundle>
+    }
 
-@{
-    string dummyImageSource = "https://dummyimage.com/1280x720/a3a3a3/fff.png?text=" + Model.BlogPost.Title;
+    <abp-script-bundle>
+        <abp-script type="typeof(HighlightJsScriptContributor)"/>
+        <abp-script src="/Pages/Public/CmsKit/highlightOnLoad.js"/>
+        <abp-script src="/Pages/Public/CmsKit/Blogs/blogpost.js"/>
```

Comment:
In `blogpost.js` it will call `Toc.init($myNav);` but `Toc.init` is defined in `bootstrap-toc.js`.  if `BlogPostScrollIndexFeature` is disabled it will run failed. 
move this line to `@if (isScrollIndexEnabled)` block?

Reply:
Thank you for your contribution @rqx110. Fixed it in [b12d8f7](https://github.com/abpframework/abp/pull/12121/commits/b12d8f71c993e34444bcffe8ae6d82be46de489d)